cmd/cli/cmd/alpha/content/terraform.go

```diff
@@ -0,0 +1,52 @@
+package content
+
+import (
+	"log"
+
+	"capact.io/capact/internal/cli"
+	"capact.io/capact/internal/cli/alpha/content"
+	"capact.io/capact/internal/cli/heredoc"
+	"github.com/spf13/cobra"
+)
+
+var tfContentCfg content.TerraformConfig
+
+// NewTerraform returns a cobra.Command to bootstrap Terraform based manifests.
+func NewTerraform() *cobra.Command {
```

Comment:
I'm wondering if it's good to generate the Terraform values as additionalInput, instead of input parameters for Interface 🤔  But then of course we would need to generate Interface along Terraform Implementation, to avoid calling Hub API.

Reply:
I wanted to split the Interface generation and Implementation generation, as you could already have an Interface and you just want to generate an Implementation for it. Also in most of our cases the Terraform variables end up in the additionalParameters.